storage/container_test.go

```diff
@@ -50,10 +50,10 @@ func TestAccContainerLifeCycle(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	if !reflect.DeepEqual(container.ReadACLs, readACLs) {
+	if diff := pretty.Compare(container.ReadACLs, readACLs); diff != "" {
 		t.Fatalf(fmt.Sprintf("ReadACLs do not match Wanted: %+v Recieved: %+v", readACLs, container.ReadACLs))
```

Comment:
Print the diff string here, instead of the list. That's the main benefit to using `pretty.Compare` 😸